helpers/helper_entitymatching.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2024 IndyKite
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"context"
+	"time"
+
+	"github.com/indykite/indykite-sdk-go/config"
+	"github.com/indykite/indykite-sdk-go/errors"
+	configpb "github.com/indykite/indykite-sdk-go/gen/indykite/config/v1beta1"
+	entitymatchingpb "github.com/indykite/indykite-sdk-go/gen/indykite/entitymatching/v1beta1"
+)
+
+type CreateAndRun struct {
+	configNodeID string
+}
+
+func (c *Client) CreateAndRunEntityMatching(
+	location string,
+	name string,
+	configuration *configpb.EntityMatchingPipelineConfig,
+	similarityScoreCutoff float32,
+) (*CreateAndRun, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	defer cancel()
+	duration, err := time.ParseDuration("10s")
+	if err != nil {
+		return nil, errors.NewInvalidArgumentErrorWithCause(err, "failed to Create duration on IndyKite Client")
+	}
```

Comment:
Nit: As the value is hardcoded probably the `err` check can be omitted and the hardcoded `10s` can be extracted in a const

Reply:
it looked strange to me but it creates an error if I don't add the err management. I will find a better way in next version